recipes/dbg-macro/all/conanfile.py

```diff
@@ -13,31 +15,25 @@ class DbgMacroConan(ConanFile):
     settings = ("compiler", )
     no_copy_source = True
```

Comment:
```suggestion
    no_copy_source = True
    package_type = "header-library"
```

Also, not a blocker at all, but if you end up having to revisit the changes, please try to remember and remove the `conan` topic entry :)

Reply:
Done, wasn't checking that, just left it as is.

Fixed. Thanks.